src/gg20/crypto_tools/k256_serde.rs

```diff
@@ -195,29 +221,28 @@ impl<'de> Deserialize<'de> for ProjectivePoint {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::sdk::implementer_api::{deserialize, serialize};
     use ecdsa::elliptic_curve::group::prime::PrimeCurveAffine;
     use k256::elliptic_curve::Field;
 
     #[test]
     fn basic_round_trip() {
         let s = Scalar(k256::Scalar::random(rand::thread_rng()));
-        let s_serialized = serialize(&s).unwrap();
-        let s_deserialized = deserialize(&s_serialized).unwrap();
+        let s_serialized = bincode::serialize(&s).unwrap();
```

Comment:
Tests should not depend on `implementer_api`.  These tests are intended only to test serde functionality so let's stick to `bincode`.